app/models/comment.rb

```diff
@@ -1,4 +1,13 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def update_comment_counter
+    post = Post.find(post_id)
+    if post.comments_counter
+      post.update(comments_counter: post.comments_counter + 1)
+    else
+      post.update(comments_counter: 1)
+    end
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think we can improve on this method based on the comments on the models above.